deploy/terraform/variables.tf

```diff
@@ -8,6 +8,11 @@ variable "project_id" {
   type        = string
 }
 
+variable "worker_count" {
```

Comment:
I haven't checked https://github.com/tinkerbell/tinkerbell.org/pull/129 to see if multiple workers will require any additional doc changes.

Reply:
No, they will just work! Nice!